src/tests/support.cpp

```diff
@@ -672,3 +672,60 @@ check_json_field_bool(json_object *obj, const std::string &field, bool value)
     }
     return json_object_get_boolean(fld) == value;
 }
+
+pgp_key_t*
+rnp_tests_get_key_by_id(const rnp_key_store_t* keyring, const std::string& keyid, pgp_key_t* after)
```

Comment:
No need to add parameter after - every call uses NULL here. And if it will be needed in future then it can be easily added.

Reply:
The 'after' is used, see 'key-store-search.cpp' lines 111, 140.

Reply:
@joke325 Right. Sorry, mislooked that place.